### DISCUSSION TITLE
Hide certain metadata

### DISCUSSION THREAD

**Discussion by pavlos2** [AzuraCast/AzuraCast#6698] [Q&A]
Hi,

I know you can edit the song's metadata one by one. Is there a setting (on the AzuraCast web interface) to stop streaming certain metadata (e.g. only the Song Artist) from all songs (no jingles) in a playlist?

Thanks in advance.

Pavlos

**Comment by BusterNeece** [comment]
Not currently, no...can you explain what the point of such a thing would be?

**Comment by pavlos2** [comment]
Hi Buster and thank you for the response. I believe there is no need to stream the metadata of all ID3 tags, too much information for the end users as well as for all those radio directory websites that like creating history playlists.
